MovieQuiz/Presentation/ AlertPresenter.swift

```diff
@@ -0,0 +1,29 @@
+//
+//   AlertPresenter.swift
+//  MovieQuiz
+//
+//  Created by Alexander Farizanov on 04.12.2022.
+//
+
+import Foundation
+import UIKit
+
+class AlertPresenter: QuestionFactoryDelegate {
+    func didRecieveNextQuestion(question: QuizQuestion?) {
+    
+    }
+    
+    private var questionFactory: QuestionFactoryProtocol? = nil
+    //questionFactory = QuestionFactory(delegate: self)
+    var alertController : UIViewController?
+    init(alertController: UIViewController? = nil) {self.alertController = alertController}
```

Comment:
Здесь надо определиться:
либо вообще в инит не передавать контроллер, а только через свойство работать, либо дефолтное значение из инита убирать совсем 

к тому же, писать в одну строчку такие иниты не принято, код выглядит очень не аккуратным
```suggestion
    init(alertController: UIViewController?) {
        self.alertController = alertController
    }
```